tests/unit/fixed-toolbar/fixedToolbar.js

```diff
@@ -2,45 +2,55 @@
  * mobile Fixed Toolbar unit tests
  */
 (function($){
-	module('jquery.mobile.toolbar.js');
+	module( "jquery.mobile.toolbar.js" );
 
-	test( "Fixed Header Structural Classes are applied correctly", function(){
+	test( "Fixed Header Structural Classes are applied correctly", function( assert ){
```

Comment:
Lets re do this to be a markup structure test ala https://github.com/jquery/jquery-ui/blob/master/tests/unit/progressbar/progressbar_core.js#L5 its very similar already.